pyfr/writers/vtkho.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from pyfr.shapes import BaseShape
+from pyfr.util import subclass_where
+from pyfr.writers.nodemaps import VTKHONodeMaps
+from pyfr.writers.vtk import VTKWriter
+
+
+class VTKHOWriter(VTKWriter):
```

Comment:
Do we want to integrate this class in with the existing VTK writer, and have it controlled through a switch?

Reply:
This could be done yes, although the switch that needs to be done with `pyr` cells which render some logic a bit ugly. Anyway, how would you inform the `vtk` writer to activate this switch from the command line? If I'm not mistaken each `writer` is associated with a unique extension and name...

Reply:
I'd use the presence of the `-d` flag to decide if to output a high order representation or to sub-divide.  So if `-d` is present, we sub-divide, otherwise we output a native high order file (which should probably be the default).

Reply:
So `-d` to subdivide and if not present use the `self.divisor = order`? I don't know if VTK scales very well with very high-order cells but we can always think about a way to reduce `self.divisor`. I'll look at it soon

Reply:
So if a divisor is not present then output a high-order VTK file.  If a divisor is provided then fall back to linear sub-divison.  For pyramids (where we have to sub-divide) then use the order as the amount of sub-division.

Reply:
You can find the modified `vtk` writer in the `wip4` commit.  Let me know if you prefer to use new logic in the `vtk` writer or if it is better to use a new writer.

Reply:
I prefer this new implementation.